routes/fire.py

```diff
@@ -0,0 +1,178 @@
+import asyncio
+from aiohttp import ClientSession
+from flask import abort, Blueprint, render_template
+from validate_latlon import validate
+from . import routes
```

Comment:
What exactly is this line doing?  Pulling in `routes` from `__init__`, ...?  Should I just read up on this?

Reply:
I made another issue to track this, so we don't have to worry about it for the moment.